contracts/token/ERC20/extensions/draft-IERC20Permit.sol

```diff
@@ -11,7 +13,7 @@ pragma solidity ^0.8.0;
  * presenting a message signed by the account. By not relying on {IERC20-approve}, the token holder account doesn't
  * need to send a transaction, and thus is not required to hold Ether at all.
  */
-interface IERC20Permit {
+interface IERC20Permit is IERC20 {
```

Comment:
I don't think this is necessary, can we remove it? I'm scared it might produce unforeseen breaking changes.

Reply:
It enables passing IERC20Permit object to functions where an IERC20 is expected.

Also, ERC2612 states
> **Specification**
> Three new functions are added to the ERC-20 ABI:

So I read that as, ERC2612 extends ERC20, [just like ERC1363](https://eips.ethereum.org/EIPS/eip-1363)

> **Specification**
> Implementing contracts MUST implement the [ERC-1363](https://eips.ethereum.org/EIPS/eip-1363) interface as well as the [ERC-20](https://eips.ethereum.org/EIPS/eip-20) and [ERC-165](https://eips.ethereum.org/EIPS/eip-165) interfaces.
>
>> interface ERC1363 /* is ERC20, ERC165 */ {

And [ERC4626](https://eips.ethereum.org/EIPS/eip-4626)

> **Specification**
> All ERC-4626 tokenized Vaults MUST implement ERC-20 to represent shares.